### ISSUE TITLE
Tests running twice...both for push and pull_request case

### ISSUE THREAD

**Issue by AndrewLane** [AndrewLane/ssh-keygen2#32]
After this change: https://github.com/AndrewLane/ssh-keygen2/pull/29, it would appear we have cases where tests are running for both the push as well as the pull_request case, now.  We need an either/or situation, instead.

**Comment by PranjalAgni** [comment]
Can you provide bit more context, as in the github action workflow config both the events are present `push` and `pull_request`, it will run for both of them.

**Comment by AndrewLane** [comment]
> Can you provide bit more context, as in the github action workflow config both the events are present `push` and `pull_request`, it will run for both of them.

Sorry, I should have been more clear with the original post.  If you look at this PR, for example, there are 29 checks which is about double I would expect.

Also, the "push" and "pull_request" runs both show up, and I really only want one or the other.  I feel like there is something fundamental I am not understanding about how the workflow runs.

![image](https://user-images.githubusercontent.com/479232/94456934-e89dd980-0181-11eb-8569-7cedaddce16d.png)

https://github.com/AndrewLane/ssh-keygen2/pull/30/checks

The reason I added pull_request in the first place is that I noticed that my checks weren't running when someone else created a pull request from their fork:https://github.com/AndrewLane/ssh-keygen2/pull/28
